charts/oes/charts/spinnaker/values.yaml

```diff
@@ -271,18 +271,6 @@ redis:
 
 # Minio access/secret keys for the in-cluster S3 usage
```

Comment:
Let's also remove this comments and add a comment saying, minio is managed by OES chart directly